### PR TITLE
[balancer-poolid] snapshot strategy + pagination

### PR DESCRIPTION
**Pull request by jo-chemla**
Snapshot Strategy for balancer
The main differences with the `balancer` strategy (from bonustrack on which this implementation is based) are:
 - the use of the `poolId` parameter, which makes the graphql query return only interesting payload.
 - and `pagination` using `skip: N`, which makes it better suited for balancer pools with more than 1.000 shares

### CODE REVIEW

File: src/strategies/index.ts [hunk 1]
```diff
@@ -460,7 +461,8 @@ const strategies = {
   'path-balance-staked-and-locked': pathBalanceStakedAndLocked,
   'sumami-holders': sUmamiHolders,
   zorro,
-  'volt-voting-power': voltVotingPower
+  'volt-voting-power': voltVotingPower, 
+  'balancer-poolId': balancerPoolId
```
Comment (ChaituVR) [inline src/strategies/index.ts:465]: Hi @jo-chemla Strategy name should be all lowercase maybe make this `balancer-poolid` or `balancer-pool-id`

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/index.ts (hunk 1 above)
Comment (jo-chemla) [reply src/strategies/index.ts]: Thanks, just did change that

File: src/strategies/index.ts [hunk 2]
```diff
@@ -230,6 +230,7 @@ import * as bottoDao from './botto-dao';
 import * as genart from './genart';
 import * as zorro from './zorro';
 import * as voltVotingPower from './volt-voting-power';
+import * as balancerPoolid from './balancer-poolid';
```
Comment (ChaituVR) [inline src/strategies/index.ts:233]: Hi @jo-chemla Rename the folder name to lower case, like here `balancer-poolid`

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/index.ts (hunk 2 above)
Comment (jo-chemla) [reply src/strategies/index.ts:233]: Ah I did that but it seems like github did not handle it

Comment (jo-chemla) [reply src/strategies/index.ts:233]: Ok that's done thanks!

**Review by bonustrack — APPROVED** [review]
Looks good

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**